lib/connectors_sdk/confluence_cloud/extractor.rb

```diff
@@ -52,7 +52,8 @@ def yield_permissions(source_user_id)
       def download(item)
         content = item[:content]
         parent_id = content.dig('container', 'id')
-        client.download("#{client.base_url}/wiki/rest/api/content/#{parent_id}/child/attachment/#{content['id']}/download").body
+        wiki_path = client.base_url.downcase.include?('/wiki') ? '' : '/wiki'
```

Comment:
what's the goal of this change? it seems unrelated to the Auth POC

Reply:
It is indeed related. For the Confluence Cloud, the base API URL is different for OAuth and API Token Auth. And it already includes `wiki` in the second case. So I had to add this to not add it again.